### PR TITLE
modularize routing and sql

### PR DESCRIPTION
**Pull request by Otis0620**
This PR cleans up the app.js file by seperating all the routes out into their own files. The PR also makes the code a little more ES6 friendly.

### CODE REVIEW

File: library/sql.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+module.exports = {
```
Comment (ybakos) [inline library/sql.js]: @Otis0620 Wouldn't it be appropriate to UP_CASE these constant names? Or is that not JavaScript-ish?

File: routes/routing-information.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
+module.exports = [
```
Comment (ybakos) [inline routes/routing-information.js]: @Otis0620 Nice catch to DRY these up. I wonder if it's best just to combine _routing-information.js_ and _misc.js_ into one file.

File: app.js [hunk 3]
```diff
@@ -1,107 +1,23 @@
-require('dotenv').config()
-const express = require('express')
-const { Pool } = require('pg')
-const app = express()
+require('dotenv').config();
+const express = require('express');
+const { Pool } = require('pg');
+const app = express();
+const { misc, rides, zagster } = require('./routes');
```
Comment (ybakos) [inline app.js]: @Otis0620 But I thought going semi-colon free was the cool modern way. :)

File: library/sql.js (hunk 1 above)
Comment (Otis0620) [reply library/sql.js]: @ybakos yah javascript is weird that way. If it is an import or destructuring then they usually stay lowercase, but if it is something like const NUM_DAYS = 11; (upper snake case) then it is uppercase.

File: routes/routing-information.js (hunk 2 above)
Comment (Otis0620) [reply routes/routing-information.js]: @ybakos isn't best to have a giant structure like routing information split out into it's own file?

File: app.js (hunk 3 above)
Comment (Otis0620) [reply app.js]: No... Always use semi colons. It's NOT cool if they are not there

**Comment by ybakos** [comment]
@Otis0620 Thanks for the PR! I merged but had to revert, there were a few missing pieces. No need to change, I appreciate the feedback!

**Pull request merged**